docs/overview/architecture.md

```diff
@@ -111,7 +111,37 @@ When the user searches a vector from Vald:
 15. Vald Egress Filter will return the filtered result to the Vald Filter Gateway.
 16. Vald Filter Gateway will return the final result to the Vald Ingress.
 
-<!-- ### Update -->
+### Update
+
+<img src="../../assets/docs/update_flow.png" />
+
+When the user updates a vector from Vald:
+
+1. Vald Ingress receives the request from the user. The request includes the the vector ID(s) and the vector(s) to be updated.
```

Comment:
```suggestion
1. Vald Ingress receives the request from the user. The request includes the new vector ID(s) and the vector(s) to be updated.
```

Reply:
```suggestion
1. Vald Ingress receives the request from the user. The request includes the the vector ID(s) and new vector(s) to be updated instead of the current indexed one.
```
 @kevindiu @hlts2 
how is it?
is it verbose? 🤔

Reply:
I think it is verbose.